clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -1013,10 +1036,10 @@ public void handle(LeaveGroupResponse leaveResponse, RequestFuture<Void> future)
 
             final Errors error = leaveResponse.error();
             if (error == Errors.NONE) {
-                log.debug("LeaveGroup request returned successfully");
+                log.debug("LeaveGroup response with {} returned successfully: {}", sentGeneration, response);
                 future.complete(null);
             } else {
-                log.error("LeaveGroup request failed with error: {}", error.message());
+                log.error("LeaveGroup response with {} failed with error: {}", sentGeneration, error.message());
```

Comment:
nit: seems the original was a little more accurate? The response did not fail; it just contained an error indicating the request had failed.